components/event-feed-service/habitat/config/config.toml

```diff
@@ -33,3 +33,8 @@ default_purge_after_days = 7
 {{~#eachAlive bind.cereal-service.members as |member|}}
 address = "{{member.sys.ip}}:{{member.cfg.port}}"
 {{~/eachAlive}}
+
+[authz]
+{{~#eachAlive bind.authz-service.members as |member|}}
```

Comment:
We could fix this separately but I think this would not be valid if we actually had more than one item in `authz-service.members`. IIRC, TOML doesn't allow you to redefine keys, so something like this would be an error:

```
address = "member1:port"
address = "member2:port"
```

Then again we probably won't have more than one item in there any time soon so no rush. But the fix is to check for `@last` like this https://github.com/chef/automate/blob/e8684cce1a1d02e95ea9429896f9793d56a37b53/components/applications-service/habitat/config/config.toml#L13-L17

Reply:
Thank you for pointing that out. I update all the services in this config file.